ui/css/design-system/_z-index.scss

```diff
@@ -3,8 +3,8 @@
 */
 $dropdown-z-index: 30;
 $main-container-z-index: 18;
-$mobile-header-z-index: 26;
```

Comment:
Removing no longer used `$mobile-header-z-index` as we are using `$header-z-index` for both